rundev.sh

```diff
@@ -4,9 +4,51 @@ NAME="dev"
 
 # Check if the container is running
 if [ "$(docker ps -a -q -f name=$NAME)" ]; then
-    echo "Stopping and removing the running container: $NAME"
-    docker stop $NAME
-    docker rm $NAME
+    # Check if the container is running
+    if [ "$(docker ps -q -f name=$NAME)" ]; then
+        echo "Container $NAME is already running."
+
+        # Prompt the user for action
```

Comment:
This might be a problem with the github actions right?
I do not want to start giving users choices after starting the script. Arguments should be okay.